Privacy.md

```diff
@@ -0,0 +1,87 @@
+# TrebleShot Privacy Policy
+This is the developer's policy on how your personal information may be collected, used, disclosed and secured. These policies are stated to ensure you know what personal details you may or may not disclose when the software is being utilized and the choices you have associated with that data. If you decide to continue using this software, it means you accept all terms and conditions stated by the developer in this policy. Kindly note that this privacy policy is restricted only to TrebleShot software and the developer is not and will not be liable to be held in account for the privacy practices of any third-party tools or services that may be contained in the software. <hr>
```

Comment:
End-users will doubt that their data is being sent away. Since it is not the case and there is no data collected in their local hardware either, could we revise it and change the parts in a way that it could help people feel secure? Thanks.

Reply:
Alright. I'll do that right away